### DISCUSSION TITLE
Design view failure issue

### DISCUSSION THREAD

**Discussion by pyking-wang** [AvaloniaUI/Avalonia#11705] [Q&A] [answered]
version：11.0.0-rc1.1
problem：In the AXAML file, whenever I make changes to the content, the previewer will malfunction. Must be recompiled before reuse.
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/89578e71-c91b-4537-adf6-63227ef66a12)
![image](https://github.com/AvaloniaUI/Avalonia/assets/76192669/448a3849-18ff-468e-bf83-e2b45bb11fae)
Hope to receive everyone's help, thank you！

**Comment by cok666n** [comment]
I have experienced the same thing in my current project. 
Is there any output from the designer that we could provide to the team that would hopefully help?

**Reply by dbriard** [reply]
Same issue here.

**Comment by pyking-wang** [comment, marked as answer]
@cok666n @dbriard hello,the issue has been resolved.refer to #11766 .using beta versions.
